src/resources/order/orders.controller.js

```diff
@@ -108,3 +112,32 @@ export const cancelOrder = async (req, res) => {
     );
   }
 };
+
+export const getAllOrders = async (req, res) => {
+  try {
+    let { page, pageSize } = req.query;
+    page = parseInt(page, 10);
+    pageSize = parseInt(pageSize, 10);
+
+    const orders = await fetchOrders(page - 1, pageSize);
+    const ordersFormat = formatAllOrdersResponse(orders.results);
+    const { previousPage, nextPage } = pagination(page, pageSize, orders);
+    const data = {
+      previousPage,
```

Comment:
What do we need prev and next page for from the backend?

Reply:
Yes, so when consuming in the frontend, you can know if there is a next page or previous page and style accordingly

Reply:
for example, disable next button when next is undefined